### PR TITLE
added renderer / fixed LG

### PR DESCRIPTION
**Pull request by ik666**
I added 2 configurations. One for the LINN App and one for the LUMIN streaming device.

Changed LG-TV-2023+ config, because `UpnpDetailsSearch` was to generic and matched other devices

### CODE REVIEW

**Comment by ik666** [comment]
Added Naim Mu-So Qb config

**Comment by SubJunk** [comment]
@ik666 what is the string from Lumin that matches the regex currently?

**Comment by ik666** [comment]
I got the following debug line

```
Matched media renderer "LG TV 2023+" based on dlna details "Wohnzimmer:LUMIN 192.168.23.31 3c393e3e-3d9b-22e2-b86c-0016e908df3b Pixel Magic Systems Ltd. U1MINI 1.0 LUMIN https://www.luminmusic.com/ https://www.luminmusic.com/"
```

**Comment by SubJunk** [comment]
Oh I see. It's matching that UUID. That's unlucky!

**Comment by ik666** [comment]
Yes ... I think some LG specific info should be taken into account like `LG.*`

File: src/main/external-resources/renderers/LG-TV-2023+.conf [hunk 1]
```diff
@@ -61,8 +61,14 @@ RendererIcon = lg-lb6500.png
...
-UserAgentSearch = \d{2}[ABCEGRWZ][3-5]|UR\d{4}
-UpnpDetailsSearch = \d{2}[ABCEGRWZ][3-5]|UR\d{4}
+
+# The following search strings are to generic and also match other devices (i.e. LUMIN from Pixel Magic).
+# 
+# UserAgentSearch = \d{2}[ABCEGRWZ][3-5]|UR\d{4}
+# UpnpDetailsSearch = \d{2}[ABCEGRWZ][3-5]|UR\d{4}
+
+UserAgentSearch = LG WebOSTV DMRplus.*\d{2}[ABCEGRWZ][3-5]|UR\d{4}
+UpnpDetailsSearch = LG WebOSTV DMRplus.*\d{2}[ABCEGRWZ][3-5]|UR\d{4}
```
[the remark this answers is not in the document]
Comment (ik666) [reply src/main/external-resources/renderers/LG-TV-2023+.conf:71]: Yes, that should work ...

**Comment by ik666** [comment]
commented out one test case that doesn't match the OLED[...] convention for the LG 2023+ profile

**Comment by ik666** [comment]
Added more specific Lumin U1 Mini configuration

@SubJunk I think it's ready now.

**Pull request merged**